pages/common/gml2gv.md

```diff
@@ -0,0 +1,17 @@
+# gml2gv
+
+> Convert a graph from the `gml` format to the `gv` format.
```

Comment:
I think that's mostly subjective, but what do you think about this:
```suggestion
> Convert a graph from `gml` to `gv` format.
```
(Same would apply to the first example)

Reply:
Writers like to use "the" and other niceties. Programmers want to save keystrokes. That's why we have `ls` instead of `list` and thousands of really short commands. grep says that tldr has more occurrences of "PDF format" etc without "the". "The" is not a heavily used word. I'll remove it and use your shorter version.

Reply:
Personally I too prefer more verbose language, but I'm also not a fan of word repititions. But only apply that if you want to. It's still your page.